packages/app/utils/api.ts

```diff
@@ -28,3 +31,49 @@ export function respondAsJson(
     // a normal response is minified JSON with the HTTP status code
     return response.status(options.httpStatusCode).json(payload)
 }
+
+async function respondAsCsv(
```

Comment:
Implemented because I had notes from our original feature meetings saying that this would be a great feature for some science users.